etc/concourse/deploy-pipeline-vars.yml

```diff
@@ -121,6 +121,9 @@ jwtalgo: algo
 # Use templates true/false
 abacus-configure: true
 
+# Slack interval
+slack: 10m
```

Comment:
I think that the default is the minimal sane value. It makes the testing easier and keeps the data as close to the month boundaries as possible.
The concrete productive slack interval varies depending on your resource providers behavior

I personally would like to keep the default as it is to avoid complicating the test setup.